docs/components.md

```diff
@@ -0,0 +1,20 @@
+# Components
+
+We divide the components into page-specific and generic components.
+
+## Page-specific components
+Place the page-specific components in the `/app` folder at the same level as the page that uses the component.
+
+## Generic components
+Components are generic if they are being used on multiple pages. Place these components in the `/components` folder.
+
+You can write generic components as *compound components* with a *dot notation*.
+
+Writing components in little parts that work together to accomplish one task is called compound components. You can read about compound components here:
```

Comment:
To make sure that I fully understand this: is the _process_ of writing components this way called 'compound components'? Or are the _results_ of writing components this way called 'compound components'?

Reply:
It's the *result*. I have changed this sentence. I hope this is more clear.